docs/src/main/asciidoc/datasource.adoc

```diff
@@ -653,6 +653,14 @@ with GraalVM native-images.
 The lack of JMX support is a natural consequence of running in native and is unlikely to be resolved.
 The integration with OCI could be resolved, if you need it we suggest opening a support request with the MySQL Connector/J maintainers.
 
+=== Oracle
+
+`jdbc:oracle:driver_type:[username/password]@database_specifier`
```

Comment:
I would drop the `[username/password]` part given we don't want it to be used like this.

`database_specifier` contains the host and port?